pkg/controller/jobframework/integrationmanager.go

```diff
@@ -78,15 +82,19 @@ func (m *integrationManager) register(name string, cb IntegrationCallbacks) erro
 	}
 
 	if cb.NewReconciler == nil {
-		return fmt.Errorf("%w \"NewReconciler\" for %q", errMissingMadatoryField, name)
+		return fmt.Errorf("%w \"NewReconciler\" for %q", errMissingMandatoryField, name)
 	}
 
 	if cb.SetupWebhook == nil {
-		return fmt.Errorf("%w \"SetupWebhook\" for %q", errMissingMadatoryField, name)
+		return fmt.Errorf("%w \"SetupWebhook\" for %q", errMissingMandatoryField, name)
 	}
 
 	if cb.JobType == nil {
-		return fmt.Errorf("%w \"WebhookType\" for %q", errMissingMadatoryField, name)
+		return fmt.Errorf("%w \"WebhookType\" for %q", errMissingMandatoryField, name)
+	}
+
+	if cb.CanSupportIntegration == nil {
```

Comment:
It shouldn't be mandatory. If you don't implement it, it means you always support it.

Reply:
Done.